### ISSUE TITLE
Remove dependance of LARA files on "LaraCore.laraGetter"

### ISSUE THREAD

**Issue by lm-sousa** [specs-feup/lara-framework#39]
The use of `LaraCore.LaraGetter` is interfering with the new JS Joinpoint wrappers

**Comment by lm-sousa** [comment]
The following block of code:

```js
console.log("q:",
    query["loop"],
    query["loop"]._javaObject,
    query["loop"].getActions()
);
```

is transpiled into:

```js
console['log']('q:',
    laraGetter(query,'loop'),
    laraGetter(laraGetter(query,'loop'),'_javaObject'),
    laraGetter(query,'loop')['getActions']()
);
```

However the code for `laraGetter` insures that any JS Joinpoint wrapper that it receives will return as `undefined` making legacy code not compatible with the new wrappers.

**Comment by lm-sousa** [comment]
Although recent changes in the `ts-conversion` branch make the `laraGetter` compatible with the access of properties, it cannot do anything about the use of methods.
The insertion of `laraGetter` should be made the exception and not the norm.

**Comment by lm-sousa** [comment]
Fixed in commit fdf6075056a56480e73ad1f0e6deaf82e53fef4b and merged in PR #41
